flipperdevices/flipperzero-firmware
# [FL-3299] furi_crash: added C2 status; added fw-version gdb command

# What's new

- [FL-3299] Now reporting C2 status and hardfault info in debug log
- Added "fw-version" gdb command extracting running firmware's version info from RTC registers

# Verification 

- Check out "fw-version" output
- Wait for c2 to crash while reading UART logs

# Checklist (For Reviewer)



[FL-3299]: https://flipperzero.atlassian.net/browse/FL-3299?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
